agent/apps/ai_guardrails/deployment/settings_additions.py

```
@@ -0,0 +1,30 @@
+"""
+تنظیمات اضافی برای ai_guardrails
+Additional Settings for ai_guardrails
+"""
+
+# اضافه کردن به INSTALLED_APPS
+INSTALLED_APPS += [
```

Ruff (0.12.2)

Undefined name `INSTALLED_APPS`

(F821)